### PR TITLE
Fixing package validation errors.

### PR DESCRIPTION
**Pull request by mmattar**
This impacts our API as two public variables have been made private.

TODO: fix our CI to catch these automatically, per commit.

### Proposed change(s)

Describe the changes made in this PR.

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [ ] Code refactor
- [X] Breaking change
- [X] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [X] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [ ] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: com.unity.ml-agents/Runtime/Sensors/CameraSensorComponent.cs [hunk 1]
```diff
@@ -63,7 +63,7 @@ public int height
         }
 
         [HideInInspector, SerializeField, FormerlySerializedAs("grayscale")]
-        public bool m_Grayscale;
+        bool m_Grayscale;
```
Comment (chriselion) [inline com.unity.ml-agents/Runtime/Sensors/CameraSensorComponent.cs:66]: Thanks, this was supposed to be private (there are public properties below)

**Comment by chriselion** [comment]
I asked on slack a few months ago about failing CI for doc xml warnings, but at the time there was nothing. If there are logs in yamato, we can parse them to do our own enforcement (similar to what I did for coverage %) but last I checked I couldn't find them.

**Comment by mmattar** [comment]
@andrewcoh - can you take a look into why the CI tests are failing? My PR doesn't touch any training code.

**Comment by andrewcoh** [comment]
> @andrewcoh - can you take a look into why the CI tests are failing? My PR doesn't touch any training code.

**Pull request merged**